packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationBillingPage.tsx

```diff
@@ -8,7 +8,7 @@ export const OrganizationBillingPage = withCardStateProvider(() => {
   const { organization } = useOrganization();
   const { data: availablePlans, isLoading: isLoadingAvailablePlans } = useFetch(
     organization?.getAvailablePlans,
-    `organization-available-plans-${organization?.id}`,
+    `organization-available-plans`,
```

Comment:
we don't need to pass organization?.id because available plans is the same for all orgs